api_tests/base/test_serializers.py

```diff
@@ -76,7 +76,7 @@ def test_registration_serializer(self):
         # fields that are visible for withdrawals
         visible_on_withdrawals = ['contributors', 'date_created', 'description', 'id', 'links', 'registration', 'title', 'type', 'current_user_can_comment', 'preprint']
         # fields that do not appear on registrations
-        non_registration_fields = ['registrations', 'draft_registrations']
+        non_registration_fields = ['registrations', 'draft_registrations', 'citation']
```

Comment:
Registrations don't get a citations relationship endpoint. I can add that, but for the purposes of pushing this forward, could that go into a follow up PR (to be worked on today)? @brianjgeiger